### PR TITLE
New version: StefHeyenrath.GitHubReleaseNotes version 1.0.9.0

### PR DESCRIPTION
**Pull request by StefH**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137624)

### CODE REVIEW

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 7:43:22 AM: 
> 2024-02-02 07:43:21.164 [FAIL] Installer failed security check. Url: https://github.com/StefH/GitHubReleaseNotes/releases/download/1.0.9.0/GitHubReleaseNotes.zip Result: 0x80004005

(Automated response - build 695.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 7:43:22 AM: 
> 2024-02-02 07:43:21.164 [FAIL] Installer failed security check. Url: https://github.com/StefH/GitHubReleaseNotes/releases/download/1.0.9.0/GitHubReleaseNotes.zip Result: 0x80004005

(Automated response - build 695.)

**Comment by stephengillie** [comment]
Hi @StefH,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 695.)

**Comment by StefH** [comment]
@stephengillie
Probably because Fody and Fody.Costura is used to include all dependencies to generate a single exe file.

**Comment by stephengillie** [comment]
Hi @StefH,

This manifest removes Apps and Features entries that are present in previous PR versions. Should these entries also be added to this version?

(Automated response - build 701.)

**Comment by StefH** [comment]
Dear @stephengillie,
Thanks for replying, can you please elaborate more on this?

My steps:
- Previously I did have a WiX setup MSI, however I could not install this because of a warning in Windows
- So I decided to only zip the exe and install the exe as portable.

File: manifests/s/StefHeyenrath/GitHubReleaseNotes/1.0.9.0/StefHeyenrath.GitHubReleaseNotes.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: StefHeyenrath.GitHubReleaseNotes
+PackageVersion: 1.0.9.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: GitHubReleaseNotes.exe
+ProductCode: '{32249977-C6C1-40ED-B6A8-7084E69A0E7A}'
+ReleaseDate: 2024-02-02
```
Comment (stephengillie) [inline manifests/s/StefHeyenrath/GitHubReleaseNotes/1.0.9.0/StefHeyenrath.GitHubReleaseNotes.installer.yaml:10]:
```suggestion
ReleaseDate: 2024-02-02
AppsAndFeaturesEntries:
- UpgradeCode: '{CA1F74E0-DFCA-4774-AE65-9421B4D08FF2}'
```
Hi @StefH,

This is only a suggestion. Other manifest versions have these lines - I'm still learning about this section and I believe they help with version matching.

**Comment by stephengillie** [comment]
Hi @StefH,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 719.)

**Comment by stephengillie** [comment]
Hi @StefH,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 731.)

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-12 15:57:40.301 [FAIL] Installer failed security check. Url: https://github.com/StefH/GitHubReleaseNotes/releases/download/1.0.9.0/GitHubReleaseNotes.zip Result: 0x80004005

(Automated response - build 731.)

**Comment by stephengillie** [comment]
Hi @StefH,

The package didn't pass a Defender or similar security scan. This might be a false positive and we can rescan tomorrow.

(Automated response - build 745.)

**Comment by stephengillie** [comment]
[Policy] Needs-Attention

**Comment by stephengillie** [comment]
[Policy] Validation-Defender-Error

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335187 2024-02-14 14:50:13.525 [FAIL] Installer failed security check. Url: https://github.com/StefH/GitHubReleaseNotes/releases/download/1.0.9.0/GitHubReleaseNotes.zip Result: 0x80004005

(Automated response - build 745.)

**Pull request closed without merging**